mmpose/models/heads/heatmap_heads/heatmap_head.py

```diff
@@ -368,7 +368,7 @@ def _load_state_dict_pre_hook(self, state_dict, prefix, local_meta, *args,
             # have keys like "final_layer.n.xxx", where the weights of the last
             # one should be renamed "final_layer.xxx", and others should be
             # renamed "conv_layers.n.xxx"
-            k_parts = _k.split('.')
+            k_parts = k.split('.')
```

Comment:
Which models are affected by this bug?

Reply:
Just wondering how HRNet survises this bug. And is HRNetv2's performance aligned with the record after fixing this or it just can load checkpoints without raising errors?